modules/ROOT/pages/_partials/sync/server-creating-a-datasource-schema-mapping.adoc

```diff
@@ -0,0 +1,137 @@
+[id="server-creating-a-datasource-schema-mapping-{context}"]
+= Creating a data source schema mapping
+
+This section describes how to create a schema in the Sync Server for the following data source mapping:
```

Comment:
use variable instead of Sync Server?
is user creating a schema _in_ the sync server?
is the following a data source mapping?